app/assets/javascripts/initializePage.js

```diff
@@ -1,7 +1,7 @@
 /*
   global initializeLocalStorageRender, initializeBodyData,
   initializeAllChatButtons, initializeAllTagEditButtons, initializeUserFollowButts,
-  initializeBaseTracking, initializeTouchDevice, initializeCommentsPage,
+  initializeBaseTracking, initializeCommentsPage,
```

Comment:
Pulled this out because this logic is now bootstrapped in `app/javascript/packs/base.js`.